### PR TITLE
{bio}[foss/2018b] samblaster v0.1.24

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/samblaster/samblaster-v.0.1.24-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#  easybuild easyconfig
+#
+#  John Dey jfdey@fredhutch.org
+#  Fred Hutchinson Cancer Research Center
+easyblock = 'MakeCp'
+
+name = 'samblaster'
+version = 'v.0.1.24'
```
Comment (boegel) [inline easybuild/easyconfigs/s/samblaster/samblaster-v.0.1.24-foss-2018b.eb]: @fizwit Should we correct this to a more standard `0.1.24`, and then change the `source_urls` and `sources` accordingly?

```python
version = '0.1.24'

...

source_urls = ['https://github.com/GregoryFaust/samblaster/releases/download/v.%(version)s']
sources = ['%(name)s-v.%(version)s.tar.gz']
```

**Review by boegel — DISMISSED** [review]

File: easybuild/easyconfigs/s/samblaster/samblaster-v.0.1.24-foss-2018b.eb (hunk 1 above)
Comment (fizwit) [reply easybuild/easyconfigs/s/samblaster/samblaster-v.0.1.24-foss-2018b.eb]: fix version and use download_filename

**Review by smoors — DISMISSED** [review]
@fizwit please remove the old file `samblaster-v.0.1.24-foss-2018b.eb ` from this PR?

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an04.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/37da019149e010536936ad32bda1a076 for a full test report.

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @fizwit!

**Pull request merged**